### DISCUSSION TITLE
Server location parameter

### DISCUSSION THREAD

**Discussion by reinhardwh** [jamulussoftware/jamulus#1665] [UI Related]
Hej
currently the "city" tells us the location of the band.

Maybe useful to share the location of the server...

In my case:
I hosted my server close to Neuss in Düsseldorf, expecting low latency.

Actually the new server located in Frankfurt is much better in respect to latency for all band members. This is maybe due to direct connection to "internet backbone"

Maybe even sharing the provider would be interesting.

**Comment by dcorson-ticino-com** [comment]
Hi Reinhard,
Note that the "city" is simply a 20 character string and you can put anything in it that you want.
Some groups use it for special sorting of the faders. 
So you can fill in any info that you would like, be it the city of the group, the city of the server, both, or anything else.

**Comment by pcar75** [comment]
As this might not be obvious to everybody  ...

One way I found to get a server location (if available) is to use "explorer.jamulus.io" site.
I try to find a list with the server name , then pick up it's IP address.  I use Google to find "Who is nnn.nnn.nnn.nnn".  After having selected an appropriate site , if it's geolocation was registered, it'll give me the city or a longitude/latitude than I use searching for "Where is xxx.xxxxxx , yyy.yyyyy".  But there might be a difference between the owner's location and the server location.
- From the "explorer.jamulus.io" entry you also get the Jamulus server version and it's OS platform (Linux for most cloud-based server or MAC / Windows for mostly owner-hosted ).
- From the 'Who is ..." search , you usually get the ISP provider (ex. AWS in the case of Amazon cloud)

Hope this helps.
